libs/judicial-system/types/src/lib/defendant.ts

```diff
@@ -1,3 +1,5 @@
+export const VERDICT_APPEAL_WINDOW_DAYS = 28
```

Comment:
Please consider moving this constant to `indictmentCase.ts` where a similar constant is defined.

Reply:
Thanks 🙏

Reply:
The constant you're referring to is `MILLISECONDS_TO_EXPIRY` and it's equal to 28 days in milliseconds. I could use that insted of creating a new 28 days constant. What do you think?

Reply:
I think you could use this new constant to calculate the old one.